README.md

```diff
@@ -22,15 +22,17 @@ Meetings are also recorded and posted to the [OpenSSF YouTube channel](https://w
 
 ## Members
 
-| Name          | Email                          | Organization |
-| ------------- | ------------------------------ | ------------ |
-| Abhishek Arya | aarya@google.com               | Google       |
-| Aeva Black    | aeva.black@microsoft.com       | Microsoft    |
-| Bob Callaway  | bcallaway@google.com           | Google       |
-| CRob Robinson | christopher.robinson@intel.com | Intel        |
-| Dan Lorenc    | dlorenc@chainguard.dev         | Chainguard   |
-| Josh Bressers | josh.bressers@anchore.com      | Anchore      |
-| Luke Hinds    | lhinds@redhat.com              | Red Hat      |
+| Name          | Email                          | Organization | Term     |
```

Comment:
```suggestion
| Name          | Email                          | Organization | Term     |
```
Should we say "Term end" here?

Reply:
I was trying to decide if it should be term start-end year like 2022-2023 or put something like "April 2022 - March 2023".  And then the line was wrapping in my text editor and I just went terse :)

Reply:
I would favour "April 2022 - March 2023". It feels like more specific is better for every use-case I can imagine.